components/engine_traits/src/tablet.rs

```diff
@@ -136,13 +136,23 @@ impl TabletContext {
     }
 }
 
+pub trait SharedResources {
```

Comment:
No need for this.

Reply:
How can get env from tablet registry?

Reply:
`Env` can be accessed through `DBOptions`. You need to update rust-rocksdb's DBOptions and tikv's DbOptions trait.